includes/blast_ui.form_advanced_options.inc

```diff
@@ -105,7 +105,7 @@ function blast_ui_blastn_advanced_options_form(&$form, $form_state) {
    $form['ALG']['SParam']['gapCost'] = array(
     '#type' => 'select',
     '#title' => t('Gap Costs:'),
-    '#options' => _get_gap('blastn'),
+    '#options' => _get_gap_options('blastn'),
```

Comment:
Renamed to avoid confusion with new function, getGap(), which returns the gap values for the selected option